contributions/catalog/webapps/templates/contribute/contribution_details.html

```diff
@@ -135,7 +135,7 @@ <h4>Talents in this contribution</h4>
         {% endfor %}
     {% endif %}
 
-    {% if reviewer %}
+    {% if (reviewer or editor or review) %}
```

Comment:
What is the difference between the `reviewer` and `review` roles in this file? We will have a user with a reviewer role who can do the review. So, as a role, I think `reviewer` will make more sense. What do you think?

Also, here the editor is probably the contribution admin who can edit the contribution, right? We will have an actual `editor` role in the future who can aggregate reviews from individual reviewers and make a final decision on the review process. It may be better to rename this to avoid confusion. Thanks.

Reply:
This is a confusing concept. 
Reviewer and editor is for deciding if the logged in user is an editor or reviewer. If the user is reviewer, the review contribution button will be activated. If the user is editor, edit contribution button will be activate. If the user is in both occasion, both button will be activated.
Review parameter is for deciding if the page should reveal the review components in the page or not. This parameter only be true when the user is reviewer in the contribut.py and rendering, so it doesn't have to be like if review == True AND if reviewer=True. 
I will rename editor to is_editable. and review to is_review, reviewer to is_reviewer, to avoid future confusion.

Reply:
renamed variables

Reply:
Thanks. I think this looks good for now.